packages/gatsby-theme-newrelic/src/components/SEO.js

```diff
@@ -21,39 +21,40 @@ const SEO = ({ title, location, children }) => {
       allLocale {
         nodes {
           locale
-          localizedPath
+          hrefLang
           isDefault
         }
       }
     }
   `);
 
-  const currentLocale = useLocale();
-
+  const locale = useLocale();
   const defaultLocale = locales.find(({ isDefault }) => isDefault);
-
   const { defaultTitle, titleTemplate, siteUrl } = siteMetadata;
-
   const template = title ? titleTemplate : '%s';
 
   const subPath =
-    currentLocale.locale === defaultLocale.locale
+    locale.locale === defaultLocale.locale
       ? location.pathname
-      : location.pathname.replace(
-          new RegExp(`\\/${currentLocale.locale}`),
-          '/'
-        );
+      : location.pathname.replace(new RegExp(`^\\/${locale.locale}`), '/');
 
   return (
     <Helmet titleTemplate={template}>
+      <html lang={locale.hrefLang} />
       <title>{title || defaultTitle}</title>
-      {locales.map(({ locale, localizedPath }, i) => {
+      <link rel="canonical" href={new URL(location.pathname, siteUrl).href} />
+      {locales.map(({ hrefLang, isDefault, locale }) => {
+        const url = new URL(
+          path.join(isDefault ? '/' : locale, subPath),
+          'https://jerelmiller.ngrok.io'
```

Comment:
I assume this was for testing :)

Reply:
Haha yep! I caught it as well. its been fixed